code/modules/projectiles/updated_projectiles/projectile.dm

```diff
@@ -424,9 +424,9 @@
 		. -= round(max(30,(shooter_human.traumatic_shock) * 0.2)) //Chance to hit declines with pain, being reduced by 0.2% per point of pain.
 		if(shooter_human.stagger)
 			. -= 30 //Being staggered fucks your aim.
-		if(shooter_human.marskman_aura)
-			. += shooter_human.marskman_aura * 1.5 //Flat buff of 3 % accuracy per aura level
-			. += P.distance_travelled * 0.35 * shooter_human.marskman_aura //Flat buff to accuracy per tile travelled
+		if(shooter_human.marksman_aura)
+			. += shooter_human.marksman_aura * 1.5 //Flat buff of 3 % accuracy per aura level
```

Comment:
didn't change the effects of focus orders since felt out of scope of PR, but focus may be giving half the intended flat accuracy bonus